testsuite/src/it/java/io/quarkus/ts/startstop/CodeQuarkusSiteTest.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.ts.startstop;
+
+import io.quarkus.ts.startstop.utils.WebpageTester;
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+/**
+ * Tests for checking presence of element on webpage
+ *
+ * @author Shpak Kyrylo <kshpak@redhat.com>
+ */
+@Tag("elementsearch")
```

Comment:
We should use `codequarkus` tag here though since this is a test for code.quarkus site

Reply:
Fix added